gammapy/utils/wcs.py

```diff
@@ -0,0 +1,133 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+"""WCS related utility functions."""
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+
+__all__ = ['linear_arrays_from_wcs',
+           'linear_wcs_from_arrays',
+           ]
+
+import numpy as np
+from astropy.wcs import WCS
+from astropy.coordinates import Angle
+
+
+def linear_arrays_from_wcs(w, nbins_x, nbins_y):
+    """Make a 2D linear binning from a WCS object.
+
+    This method gives the correct answer only for linear X, Y binning.
+    The method expects angular quantities in the WCS object.
+    X is identified with WCS axis 1, Y is identified with WCS axis 2.
+    The method needs the number of bins as input, since it is not in
+    the WCS object.
+
+    Parameters
+    ----------
+    w : `~astropy.wcs.WCS`
```

Comment:
Ok. I was using `w`, because I have to call the `wcs` method, so `wcs.wcs` seemed too much. But I did as you suggested.

I updated the method names, for the second time :-)